### ISSUE TITLE
 Get-PASAccount -filter is not returning any results but is receiving body

### ISSUE THREAD

**Issue by AndrewCopeland** [pspete/psPAS#304]
**Describe the issue**
Currently if I perform just `Get-PASAccount` I can retrieve all the accounts inside of Cyberark. However when I attempt to filter on `SafeName` I am not receiving any accounts.

**To Reproduce**
1.  Install Module
2.  Import Module
3.  Create new pssession
4. then run `Get-PASAccount -filter "SafeName eq safeName"

**Expected behavior**
I expect to a get a list of accounts that exists within the safe provded.

Console Output Code Block:
```powershell
PS C:\Users\andco> $VerbosePreference = "continue"
PS C:\Users\andco> Get-PASAccount -filter "SafeName eq WindowsDualAccount"
VERBOSE: GET https://[redacted]/PasswordVault/api/Accounts?filter=SafeName eq
WindowsDualAccount with 0-byte payload
VERBOSE: received 1059-byte response of content type application/json; charset=utf-8
PS C:\Users\andco> Get-PASAccount -filter "SafeName eq DevOps_Safe"
VERBOSE: GET https://[redacted]/PasswordVault/api/Accounts?filter=SafeName eq
DevOps_Safe with 0-byte payload
VERBOSE: received 2663-byte response of content type application/json; charset=utf-8
PS C:\Users\andco> Get-PASAccount -filter "SafeName eq NotReal"
VERBOSE: GET https://[redacted]/PasswordVault/api/Accounts?filter=SafeName eq NotReal
with 0-byte payload
VERBOSE: received 22-byte response of content type application/json; charset=utf-8
```

**Your Environment**
Include relevant details about your environment

* PowerShell Version: 5.1
* psPAS Version: v4.3.62
* CyberArk Version: 10.8

**Additional context**

**Comment by pspete** [comment]
🙏 Thanks for reporting @AndrewCopeland - this is related to a recent update to the internal logic  of`Get-PASAccount` - missed this before it made it into the release.
Fix identified & incoming.
In the meantime, you should be able to get the results like this:

```powershell
Get-PASAccount -SafeName WindowsDualAccount
```

**Comment by pspete** [comment]
Fixed in `psPAS 4.3.65`
